twake/backend/node/src/services/previews/services/processing/pdf.ts

```diff
@@ -0,0 +1,26 @@
+import { PDFImage } from "pdf-image";
+import { unlink } from "fs/promises";
+
+export async function convertFromPdf(
+  inputPath: string,
+  numberOfPages: number,
+): Promise<{ output: string[]; done: boolean }> {
+  let pages: string[] = [];
+
+  try {
+    const pdfImage = new PDFImage(inputPath);
+
+    numberOfPages = 1; //Fixme: We cannot generate more page because of a library bug right now
```

Comment:
which bug? Did you create an issue on github? If yes, put the link in the comment please